interpreter/cling/test/Interfaces/evaluate.C

```diff
@@ -44,13 +44,13 @@ V // CHECK: (cling::Value &) boxes [(long) 17]
 
 int* IntP = (int*)0x12;
 gCling->evaluate("IntP;", V);
-V // CHECK: (cling::Value &) boxes [(int *) 0x12]
+V // CHECK: (cling::Value &) boxes [(int *) 0x12 INVALID]
```

Comment:
Can we rename this to `<invalid memory address>`? We cannot really assert valid memory addresses...